src/styles/utilities/cursor/types.ts

```diff
@@ -13,6 +13,6 @@ type CursorStylesKeys =
     | 'cursorInitial'
     | 'cursorText';
 
-type CursorStyles = Record<CursorStylesKeys, Partial<Pick<CSSProperties, 'cursor'>>>;
+type CursorStyles = Record<CursorStylesKeys, Pick<ViewStyle, 'cursor'>>;
```

Comment:
```suggestion
type CursorStyles = Record<CursorStylesKeys, ViewStyle['cursor']>;
```

When there is only one property to `Pick` from an existing type, you can just do something like above

Reply:
Similar to above comment